File: Week 1 (C)/Lecture 1/Agree.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    char x = get_char("Do you agree?: ");

    if (x == 'y' || x == 'Y')
    {
        printf("Agreed\n");
    }
    else if (x == 'n' || x == 'N')
    {
        printf("Not agreed\n");
    }
    else
    {
        printf("Error. Please answer with Y/N");
    }
}

File: Week 6 (Python)/Lecture 6/Agree.py
answer = input("Do you agree?: ")

# if answer == Y or answer == y or answer == Yes or answer == yes or answer == YES:
if answer.lower() in ["y", "yes"]:
    print("Agreed")
# elif answer == N or answer == n or answer == No or answer == no or answer == NO:
elif answer.lower() in ["n", "no"]:
    print("Not agreed")
else:
    print("Huh?")